binder/Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 # Tag will be automatically generated through pre-commit hook if any changes
 # happened in the docker/ folder
-FROM ghcr.io/projectnessie/nessie-binder-demos:7fb962b09182c5f60ac2f80bbabb693290f3f741
+FROM ghcr.io/projectnessie/nessie-binder-demos:788b8cd170f0a67371e200dab75be3f97fd20d74
```

Comment:
i ran the script to update this tag locally... but it wont built until this PR is merged an the workflow for `main` builds the latest images and publishes them to `ghcr`